### PR TITLE
Restore the website workflow to the AWS getting-started guide

### LINKED ISSUE #2720: Restore the bucket-website flow to the getting-started guide
With the fixes in https://github.com/pulumi/pulumi-hugo/pull/2719, we removed (because it [suddenly stopped working](https://github.com/pulumi/pulumi-hugo/issues/2718)) a good portion of the tutorial that illustrated some important Pulumi concepts:

* Modifying the properties of a resource (bucket -> bucket website, private -> public object)
* Exporting additional stack outputs (website endpoint)
* Verifying the deployment (browsing to the website URL)

If possible, we should restore as much of this experience as we can, as soon as we can -- ideally without making it overly complicated for new users. 

For reference, the code to make this work, given the new restrictions, would look something like this (in TS -- we'd of course need to translate and verify across all other languages):

```typescript
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const bucket = new aws.s3.Bucket("my-bucket", {
    website: {
        indexDocument: "index.html",
    },
});

// 👇 New
const ownershipControls = new aws.s3.BucketOwnershipControls("ownership-controls", {
    bucket: bucket.id,
    rule: {
        objectOwnership: "ObjectWriter"
    }
});

// 👇 New
const publicAccessBlock = new aws.s3.BucketPublicAccessBlock("public-access-block", {
    bucket: bucket.id,
    blockPublicAcls: false,
});

const bucketObject = new aws.s3.BucketObject("index.html", {
    acl: "public-read",
    contentType: "text/html",
    bucket: bucket.id,
    source: new pulumi.asset.FileAsset("index.html")
}, { dependsOn: ownershipControls });                   // 👈 New

export const bucketName = bucket.id;
export const bucketEndpoint = pulumi.interpolate`http://${bucket.websiteEndpoint}`;
```

Related to https://github.com/pulumi/pulumi-aws/issues/2470.

### PR DESCRIPTION
**Pull request by cnunciato**
## Description

Fixes #2720.

The code changes here are based on a new repo, [pulumi/getting-started](https://github.com/pulumi/getting-started), that I've set up to test that each step of the guide continues to work as expected.

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [x] I added aliases (i.e., redirects) for all filename changes.

### CODE REVIEW

File: themes/default/content/docs/get-started/aws/deploy-changes.md [hunk 1]
```diff
@@ -117,10 +115,426 @@ $ aws s3 ls $(pulumi stack output bucketName)
...
+    bucket: bucket.id,
+    blockPublicAcls: false,
+});
+
+const bucketObject = new aws.s3.BucketObject("index.html", {
+    bucket: bucket.id,
+    source: new pulumi.asset.FileAsset("index.html"),
+    contentType: "text/html",
+    acl: "public-read",
+}, { dependsOn: ownershipControls });
```
Comment (scottslowe) [inline themes/default/content/docs/get-started/aws/deploy-changes.md:236]: All the other code examples specify the public access block for `DependsOn`, should this one match the others?

**Review by scottslowe — CHANGES_REQUESTED** [review]
Looking good, just one minor catch

File: themes/default/content/docs/get-started/aws/deploy-changes.md (hunk 1 above)
Comment (cnunciato) [reply themes/default/content/docs/get-started/aws/deploy-changes.md:236]: Yes -- good catch!

**Comment by susanev** [comment]
im unsure if im going to be able to look at this today, but i will try. don't block on me though!!

File: themes/default/content/docs/get-started/aws/deploy-changes.md [hunk 2]
```diff
@@ -117,12 +116,494 @@ $ aws s3 ls $(pulumi stack output bucketName)
...
+  properties:
+    website:
+      indexDocument: index.html
+```
+
+{{% /choosable %}}
+
+Lastly, you'll make a few adjustments to make these resources accessible on the Internet.
+
+For the bucket itself, you'll need two new resources: a `BucketOwnershipControls` resource, to define the bucket's file-ownership settings, and a `BucketPublicAccessBlock` resource to allows the bucket to be accessed publicly.
```
Comment (scottslowe) [inline themes/default/content/docs/get-started/aws/deploy-changes.md:217]:
```suggestion
For the bucket itself, you'll need two new resources: a `BucketOwnershipControls` resource, to define the bucket's file-ownership settings, and a `BucketPublicAccessBlock` resource to allow the bucket to be accessed publicly.
```

File: themes/default/content/docs/get-started/aws/destroy-stack.md [hunk 3]
```diff
@@ -25,33 +25,38 @@ You'll be prompted to make sure you really want to delete these resources. This
...
 
 Resources:
-    - 3 deleted
+    - 5 deleted
 
-Duration: 3s
+Duration: 4s
 ```
 
 > To delete the stack itself, run [`pulumi stack rm`](/docs/reference/cli/pulumi_stack_rm). Note that this removes the stack entirely from the Pulumi Cloud, along with all of its update history.
```
Comment (scottslowe) [inline themes/default/content/docs/get-started/aws/destroy-stack.md:62]:
```suggestion
> To delete the stack itself, run [`pulumi stack rm`](/docs/reference/cli/pulumi_stack_rm). Note that this removes the stack entirely from Pulumi Cloud, along with all of its update history.
```

**Review by scottslowe — APPROVED** [review]
Approved but a couple of suggestions for you to consider

**Pull request merged**